src/Components/Header/Navbar.css

```diff
@@ -0,0 +1,160 @@
+.header {
+	position: fixed;
+	top: 0;
+	left: 0;
+	right: 0;
+	background: var(--bg-color-1);
+	box-shadow: var(--box-shadow);
+	padding: 1.5rem 7%;
+	z-index: 1000;
+	display: flex;
+	align-items: center;
+	justify-content: space-between;
+}
+
+.header .logo {
+	font-weight: bolder;
+	font-size: 2.5rem;
+	color: var(--text-color-1);
+}
+
+.header .logo i {
+	color: var(--primary);
+	font-size: 48px;
+	transform: rotate(-45deg);
+}
+
+.header .search-form {
+	background: var(--bg-color-2);
+	border-radius: .5rem;
+	display: flex;
+	align-items: center;
+	height: 4.5rem;
+	width: 50rem;
+}
+
+.header .search-form input {
+	height: 100%;
+	width: 100%;
+	background: none;
+	text-transform: none;
+	font-size: 1.5rem;
+	color: var(--text-color-1);
+	padding: 1rem;
+}
+
+.header .search-form label {
+	font-size: 2rem;
+	margin-right: 1.5rem;
+	color: var(--text-color-1);
+	cursor: pointer;
+}
+
+.header .search-form label:hover {
+	color: var(--primary);
+}
+
+.header .icons div {
+	height: 4.5rem;
+	width: 4rem;
+	line-height: 4.5rem;
+	font-size: 1.5rem;
+	border-radius: .5rem;
+	margin-left: .2rem;
+	background: var(--bg-color-2);
+	color: var(--text-color-1);
+	cursor: pointer;
+	text-align: center;
+}
+
+.header .icons div:hover {
+	color: #fff;
```

Comment:
Avoid using like this use predefined color or else predefine it also in the root file.